### PR TITLE
Added test file for edit profile page

### LINKED ISSUE #1522: Test edit_profile_page.dart
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file:
```
lib/views/after_auth_screens/profile/edit_profile_page.dart
```
- When complete, all methods, classes and/or functions in the file will need to be tested. 
- Our `test/` folder has the same structure as the `lib` folder. Place your test file in the equivalent folder under `test/`. You may need to create the appropriate directory structure to do this.
- [Current test code coverage for this file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa?search=&displayType=list)

### IMPORTANT:
Please refer to this issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)

### PR DESCRIPTION
**Pull request by Insiyaya**
Added test file for edit profile page with required testing classes and widgets

**What kind of change does this PR introduce?**

**Issue Number:** #1522 

Fixes #1522 

**Did you add tests for your changes?**

 Yes

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

 Test case file for edit profile page

**Does this PR introduce a breaking change?**  No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

 Yes

### CODE REVIEW

**Comment by literalEval** [comment]
@Insiyaya please format the code properly before opening PRs. Thanks.

**Comment by Cyclotron17** [comment]
Hey , can you please assign this issue to me.

**Comment by Insiyaya** [comment]
> @Insiyaya please format the code properly before opening PRs. Thanks.

Hey can you brief what changes i have to make in the code..It will be easy then

**Comment by literalEval** [comment]
@Insiyaya you need to run "dart format --set-exit-if-changed ." before commiting. This is mentioned in Contribuing.md

**Comment by Insiyaya** [comment]
Okay


> @Insiyaya you need to run "dart format --set-exit-if-changed ." before commiting. This is mentioned in Contribuing.md

Okay

**Comment by Insiyaya** [comment]
> @Insiyaya you need to run "dart format --set-exit-if-changed ." before commiting. This is mentioned in Contribuing.md

Okay

**Comment by Insiyaya** [comment]
I have formatted the code and pushed it

File: test/views/after_auth_screens/profile/edit_profile_page_test.dart [hunk 1]
```diff
@@ -0,0 +1,226 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:talawa/view_model/after_auth_view_models/profile_view_models/edit_profile_view_model.dart';
+import 'package:talawa/views/after_auth_screens/profile/edit_profile_page.dart'; // Import the EditProfilePage widget
```
Comment (literalEval) [inline test/views/after_auth_screens/profile/edit_profile_page_test.dart:5]: Why is there a comment for the import ?

File: test/views/after_auth_screens/profile/edit_profile_page_test.dart [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+void main() {
+  group(
+    'EditProfilePage Widget Tests',
+    () {
+      testWidgets(
+        'Widget renders and updates correctly',
+        (WidgetTester tester) async {
+          final mockEditProfileViewModel = MockEditProfileViewModel();
+
+          when(mockEditProfileViewModel.initialize()).thenAnswer((_) async {});
```
Comment (literalEval) [inline test/views/after_auth_screens/profile/edit_profile_page_test.dart:20]: You've done this mock injection too many times. This violates the DRY principle and should be avoided. Do all of the injection and initialization during setupAll()

**Review by literalEval — CHANGES_REQUESTED** [review]

File: test/views/after_auth_screens/profile/edit_profile_page_test.dart (hunk 2 above)
Comment (Insiyaya) [reply test/views/after_auth_screens/profile/edit_profile_page_test.dart:20]: Okay

**Review by literalEval — APPROVED** [review]
LGTM

**Review by literalEval — CHANGES_REQUESTED** [review]
One of the test is failing. Fix it.

**Comment by Dante291** [comment]
@Insiyaya why new test file for edit_profile_page? There's already a test file for this page [here](https://github.com/PalisadoesFoundation/talawa/blob/develop/test/widget_tests/after_auth_screens/profile/edit_profile_page_test.dart).

Currently the coverage for this file is 95% you just have to add more tests so that coverage get to 100%, you can check the current coverage for this file [here](https://app.codecov.io/gh/PalisadoesFoundation/talawa/blob/develop/lib%2Fviews%2Fafter_auth_screens%2Fprofile%2Fedit_profile_page.dart).

**Comment by Insiyaya** [comment]
Mentors Please Review my recent PR

**Comment by Insiyaya** [comment]
hey mentors, can you explain me or help me figure out what is the issue in the file? it is failing checking codebase test

**Comment by palisadoes** [comment]
Please read the error, it is explained there

**Comment by Insiyaya** [comment]
> Please read the error, it is explained there
Yes i checked it and this is the error...but i have not changed anything in these files given below..so i am not understanding basically what to do..plwase help me figure that out..thanks :)
[Ignore directive for custom lint rule found. Please remove them and add valid documentation.
>>> File name: lib/models/organization/org_info.g.dart

>>> File name: lib/views/pre_auth_screens/waiting_to_join_private_org.dart...]

**Comment by Azad99-9** [comment]
@Insiyaya even though you didn't change any related code, it's a good practice to remove ignore directives and add docs to specified files. Please add docs to get your PR reviewed.

**Comment by Dante291** [comment]
Also why are you removing profile_page_test.dart file?

**Comment by Insiyaya** [comment]
> Also why are you removing profile_page_test.dart file?

it was previously added by me cuz of some misconceptions

**Comment by palisadoes** [comment]
@Insiyaya 

We have a policy of unassigning contributors who close PRs without getting validation from our reviewer team. This is because:

1. We start looking for people to review PRs when you submit them. 
2. We often contact them and link to the PR. If the PR is closed the whole effort is wasted.
3. The historical thread of reviewer comments is broken when the work is spread across multiple PRs. The quality of our code is affected negatively.

Please be considerate of our volunteers' limited time and our desire to improve our code base.

This policy is stated as a pinned post in all our Talawa repositories.  

We value all contributions to the repository, but the frequency of this type of activity has forced our hand. 

We will enforce this policy the next time.

**Pull request closed without merging**